### PR TITLE
Focus menu item

### PR DESCRIPTION
**Pull request by stegru**
- Removes the standard mode menu item when using a mobile
- Always showing the _Person Details_ link when in focus/mobile mode

### CODE REVIEW

**Review by christopher-rtf — CHANGES_REQUESTED** [review]
The following scenario doesn't work yet; please update this PR to include the following fix.

Scenario:
1. Go to the web app in a desktop browser (desktop-width)
2. Click on "Focus/Mobile" button in top-right
3. Shrink the width of the browser until "Standard Mode" is no longer possible
4. The hamburger menu still has a "Standard Mode" option; it should not...as clicking it does nothing for the user.

This bug probably also presents itself if the user simply shrinks their desktop browser to "mobile" width...telling them that they can still switch back to "Standard Mode".

TL;DR: the "Standard Mode" button should only appear in the hamburger menu when the browser is wide enough for it.

**Review by christopher-rtf — APPROVED** [review]
